src/client/Microsoft.Identity.Client/AuthScheme/IAuthenticationOperation.cs

```diff
@@ -9,12 +9,28 @@ namespace Microsoft.Identity.Client.AuthScheme
     /// <summary>
     /// Used to modify the experience depending on the type of token asked. 
     /// </summary>
-    internal interface IAuthenticationScheme
+    public interface IAuthenticationOperation
     {
         /// <summary>
-        /// Value to log to telemetry to indicate pop usage.
+        /// Value to log to telemetry
+        /// Values available:
```

Comment:
We don't need to maintain these here? if `public int TelemetryTokenType => (int)TokenType.Pop;`

Reply:
We are going to change how this value works in the future. Most likely make it a string and modify the expected values.